src/nisarqa/products/rslc.py

```diff
@@ -1328,9 +1347,7 @@ def generate_backscatter_image_histogram_single_freq(
     """
     log = nisarqa.get_logger()
 
-    log.debug(
-        f"Generating Backscatter Image Histograms for Frequency {freq}..."
-    )
+    log.info(f"Generating Backscatter Image Histograms for Frequency {freq}...")
```

Comment:
The "..." seems to imply that there will be a log message later saying we've finished with the backscatter histograms, but that doesn't seem to exist.

Also, there's no similar log message for the phase histograms.

Should we remove this log message? Or else add in the missing log messages?

Reply:
Added missing log messages, and set them all to logging level of "info".